### PR TITLE
Revamp parse JSON

### PR DESCRIPTION
**Pull request by kigawas**
#2 

- uv to manage dependencies
- pydantic-xml to parse xml
- remove redudant files

### CODE REVIEW

File: Apuleius/opensource/apuleius.ga_lat.xml.json [hunk 1]
```diff
@@ -1 +0,0 @@
...itior cerva, sed aper immanis atque invisitatus exsurgit toris callosae cutis obesus, pilis inhorrentibus corio squalidus, setis insurgentibus spinae hispidus dentibus attritu sonaci spumeus, oculis aspectu minaci flammeus, impetu saevo frementis oris totus fulmineus; et primum quidem canum procaciores quae comminus contulerant vestigium, genis hac illac iactatis consectas interficit, dein calcata retiola,  qua primos impetus reduxerat, transabiit. Et nos quidem cuncti pavore deterriti et alioquin innoxiis venationibus consueti, tunc etiam inermes atque immuniti, tegumentis frondis vel arboribus latenter abscondimus; Thrasyllus vero nanctus fraudium opportunum decipulum sic Tlepolemum captiose compellat: ; nec tantillum morati protinus insiliunt equos ex summo studio bestiam insequentes. Nec tamen illa genuini vigoris oblita retorquet imi petum et incendio feritatis ardescens dente compulsa quem primum insiliat cunctabunda rimatur. Sed prior Tlepolemus iaculum, quod gerebat, insuper dorsum bestiae contorsit: at Thrasyllus ferae quidem pepercit sed equi, quo vehebatur Tlepolemus, postremos poplites lancea feriens amputat. Quadrupes reccidens, qua sanguis effluxerat, toto tergo supin
```
Comment (kylepjohnson) [inline Apuleius/opensource/apuleius.ga_lat.xml.json:1]: @kigawas In the GH PR, it reads "This file was deleted." -- was this intentional? There are >700 file changes, so it's hard for me to see what all has been altered. Thx

**Comment by kylepjohnson** [comment]
> remove redudant files

So you removed the files ending with `.xml.json`, but kept the rest? TBH I do not remember why those were in there. Is the formatting/content of those any better than those with just `.json`?

File: Apuleius/opensource/apuleius.ga_lat.xml.json (hunk 1 above)
Comment (kigawas) [reply Apuleius/opensource/apuleius.ga_lat.xml.json:1]: You can use file filter to hide .json and .xml files

**Comment by kigawas** [comment]
.xml.json file was probably just converted from its corresponding xml file. You may have thought it's easier to process json or something.

Since it has neither more nor less information than xml so it's better to just remove. We can directly handle xml by the lxml library

**Comment by kylepjohnson** [comment]
> .xml.json file was probably just converted from its corresponding xml file. You may have thought it's easier to process json or something.

Yes it was, but we do not want to remove any of these. There was a concerted effort to transform all those XML into JSON that they be more readable by developers of web applications. So whatever we do, cannot delete the JSON files.

I would like to close this PR and for your to start fresh. We do need better parsing of the XML (and you showed our previous work was not good enough) however we must parse it for some purpose, and into some format (we need to keep the current JSON schema). Thanks in advance for your understanding, we can keep talking here.

**Pull request closed without merging**